src/Parser/BloodDeathKnight/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellLink from 'common/SpellLink';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static ABILITIES_ON_GCD = [
+    SPELLS.CONSUMPTION.id,
+    SPELLS.DEATH_AND_DECAY.id,
+    SPELLS.BLOOD_BOIL.id,
+    SPELLS.HEART_STRIKE.id,
+    SPELLS.MARROWREND.id,
+    SPELLS.DEATH_STRIKE.id,
+    SPELLS.DEATHS_CARESS.id,
+    SPELLS.BLOODDRINKER.id,
+    //CDS
+    SPELLS.ICEBOUND_FORTITUDE.id,
+    SPELLS.DANCING_RUNE_WEAPON.id,
+    SPELLS.VAMPIRIC_BLOOD.id,
+    SPELLS.Anti_Magic_Shell.id,
```

Comment:
This and Asphyxiate should also use capital letters.

Reply:
Fixed